package.json

```diff
@@ -202,7 +202,7 @@
     "ignore-sync": "^5.0.1",
     "lint-staged": "^13.0.3",
     "mocha": "^10.0.0",
-    "prettier": "^2.4.1",
+    "prettier": "^2.7.1",
```

Comment:
I tried updating it because the formatter was just breaking a certain code construct by inserting a random semicolon. I couldn't reproduce it in the prettier REPL, and ended up just changing the code.

We apparently don't include format in CI anymore, because (1) there were a couple of files that weren't formatted, and (2) there were a couple of generated files that were errantly not being excluded from lint (`npm run format` runs lint first, for some reason). Outside of those changes, I did run format and it doesn't change anything.